external/fv3fit/tests/sklearn/test_transformer.py

```diff
@@ -0,0 +1,96 @@
+from fv3fit.sklearn.transformer import SkTransformer
+import numpy as np
+import os
+from sklearn.preprocessing import StandardScaler
+
+
+class DummyTransformer:
+    def __init__(self, latent_dim, output_dim, encoded_constant, decoded_constant):
+        self.latent_dim = latent_dim
+        self.output_dim = output_dim
+        self.encoded_constant = encoded_constant
+        self.decoded_constant = decoded_constant
+
+    def transform(self, X):
+        return np.ones(self.latent_dim) + self.encoded_constant
+
+    def inverse_transform(self, X):
+        return np.ones(self.output_dim) + self.decoded_constant
+
+
+def test_sktransformer_roundtrip():
+    latent_dim = 5
+    output_dim = 10
+    n_vars = 3
+    encoded_constant = 0
+    decoded_constant = -2
+
+    scaler = StandardScaler()
+    scaler.fit(np.random.rand(5, n_vars * output_dim))
+    transformer = transformer = DummyTransformer(
+        latent_dim, n_vars * output_dim, encoded_constant, decoded_constant
+    )
+    sktransformer = SkTransformer(transformer, scaler, enforce_positive_outputs=False)
+
+    x = [np.random.rand(output_dim) for i in range(n_vars)]
+    target = np.ones(n_vars * output_dim) + decoded_constant
+    outputs = sktransformer.predict(x)
```

Comment:
As it currently stands, there's no true test of the outputs of the `encode` method used in `predict` roundtrip since the `DummyTransformer` doesn't use the input `X`.